### DISCUSSION TITLE
Wrong ISO code for the Kurdish language translations

### DISCUSSION THREAD

**Discussion by halwesit** [directus/directus#9786] [Archive]
Hello Everyone.

A couple of days ago I tried to translate the Directus into the Kurdish Language, and I did some, but then I noticed that the Kurdish Language ISO code is wrong and it was written as [ku_TR ](https://github.com/directus/directus/blob/main/app/src/lang/translations/ku-TR.yaml) and there's no such ISO code for the Kurdish language, So let's explain how is the Kurdish language.

The [Kurdish Language](https://en.wikipedia.org/wiki/Kurdish_languages) has three main dialects (Sorani, Kurmanji and Southern) all have different ISO codes, because of the deference of writing scripts. the below list is shown the name, ISO code and the writing script for each dialect as well as the reference of their ISO codes.

- Sorani AKA Central Kurdish - ISO code: [CKB](https://iso639-3.sil.org/code/ckb), written in Arabic script
- Kurmanji AKA Northern Kurdish - ISO code: [KMR](https://iso639-3.sil.org/code/kmr), written in English script
- Southern Kurdish - ISO code: [SDH](https://iso639-3.sil.org/code/sdh), written in Arabic script

Please, review this issue, and we will appreciate renaming the current added translation into CKB and adding others as well.

Thanks

Halwest Manguri

**Comment by rijkvanzanten** [comment]
Heya!

Thanks for the heads up. The language codes are currently provided by Crowdin (https://locales.directus.io) directly, so I'll move this particular concern over to their support staff 👍🏻 

In the meantime, what would you recommend we use for the language code for the current translations?

**Reply by rijkvanzanten** [reply]
I checked in with Crowdin support. They're treating SDH (southern kurdish) as "Kurdish" w/ lang code `ku`, and have additional languages (w/ codes) available for Kurmanji `kmr` and Sorani `ckb`. This seems to align with ISO 639-1. 

Were you looking to add translations for either of the other two dialects? We can enable that from our crowdin settings 👍🏻

**Reply by halwesit** [reply]
Thanks for your reply, for the current translations, please use `CKB`, because it's in Sorani "Central Kurdish". and for the Southern Kurdish if you use `ku` please mention that this ISO code is for the Southern Kurdish.

thanks

**Reply by halwesit** [reply]
Hey @rijkvanzanten, I'm still waiting for the changes.

**Reply by halwesit** [reply]
And also the Kurdish translations missed in the [available translations file](https://github.com/directus/directus/blob/main/app/src/lang/available-languages.yaml)
